gallery/src/styles/NxPageTitle/NxPageTitlePage.tsx

```diff
@@ -7,28 +7,32 @@
 import React from 'react';
 import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
 import { GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+import { NxP, NxCode } from '@sonatype/react-shared-components';
 
 import NxPageTitlePolicyViolationIndicatorExample from './NxPageTitlePolicyViolationIndicatorExample';
+import NxPageTitleEverythingExample from './NxPageTitleEverythingExample';
 
 const nxPageTitleCode = require('./NxPageTitleExample.html'),
     nxPageTitleActionsCode = require('./NxPageTitleActionsExample.html'),
+    nxPageTitleSubtitleCode = require('./NxPageTitleSubtitleExample.html'),
     nxPageTitlePolicyViolationIndicatorCode =
-      require('./NxPageTitlePolicyViolationIndicatorExample.tsx?raw');
+      require('./NxPageTitlePolicyViolationIndicatorExample.tsx?raw'),
+    nxPageTitleEverythingCode = require('./NxPageTitleEverythingExample.tsx?raw');
 
 const NxPageTitlePage = () =>
   <>
     <GalleryDescriptionTile>
-      <p className="nx-p">
+      <NxP>
         <code className="nx-code">nx-page-title</code> is used at the top of a page, it always has a title, and can also
-        have an icon and descriptive text.
-      </p>
-      <p className="nx-p">
-        In addition <code className="nx-code">.nx-page-title</code> can have tags or actions inline with the title.
-      </p>
-      <p className="nx-p">
+        have a sub-title, descriptive text, and space for tags or other indicators.
+      </NxP>
+      <NxP>
+        In addition <code className="nx-code">.nx-page-title</code> can have buttons inline with the title.
+      </NxP>
+      <NxP>
         Note: <code className="nx-code">.nx-page-title</code> replaces
         <code className="nx-code">.nx-tile--top-tile</code> and <code className="nx-code">.nx-tile--title-only</code>.
-      </p>
+      </NxP>
       <table className="nx-table nx-table--gallery-props">
```

Comment:
Not from this PR, but the second row in this table is missing a cell

Reply:
> Not from this PR, but the second row in this table is missing a cell

That whole row should have been removed. @dturner32 requested that we remove the icon from page headers. Also the class was never defined in the SCSS.